<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSetting</class>
 <widget class="QDialog" name="DialogSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>671</width>
     <height>341</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ventap</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>POS 1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPosHost1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.0.1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPosAddress1">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>321</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>/var/lib/firebird/2.5/data/ventap.fdb</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPosHost2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.0.2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>POS 2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPosAddress2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>321</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>/var/lib/firebird/2.5/data/ventap.fdb</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPosHost3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.0.3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>POS 3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPosAddress3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>321</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>/var/lib/firebird/2.5/data/ventap.fdb</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPosHost4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.0.4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>POS 4</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPosAddress4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>321</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>/var/lib/firebird/2.5/data/ventap.fdb</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPosHost5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.0.5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>POS 5</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPosAddress5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>321</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>/var/lib/firebird/2.5/data/ventap.fdb</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPosHost6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.0.6</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>POS 6</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPosAddress6">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>240</y>
      <width>321</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>/var/lib/firebird/2.5/data/ventap.fdb</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPosHost7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.0.7</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>POS 7</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPosAddress7">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>290</y>
      <width>321</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>/var/lib/firebird/2.5/data/ventap.fdb</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConfirm">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>520</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>671</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ERP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditErpHost">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.0.7</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditErpAddress">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>321</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>/var/lib/firebird/2.5/data/ventap.fdb</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
